src/hecs/frag_iter.rs

```diff
@@ -23,7 +23,7 @@ impl Benchmark {
     }
 
     pub fn run(&mut self) {
-        for (_, mut data) in self.0.query_mut::<&mut Data>() {
+        for (_, mut data) in self.0.query::<&mut Data>().iter() {
```

Comment:
This change is not related to an update, and should not be required.

Reply:
Yeah, this one slipped in accidentally.